ql/src/java/org/apache/hadoop/hive/ql/exec/GroupByOperator.java

```diff
@@ -411,13 +412,12 @@ protected void initializeOp(Configuration hconf) throws HiveException {
    *         aggregation only
    **/
   private void computeMaxEntriesHashAggr() throws HiveException {
```

Comment:
can we pass `maxMemory` as a param here to avoid reordering issues after refactor in future?